lib/gooddata/models/user_filters/user_filter_builder.rb

```diff
@@ -261,22 +261,22 @@ def self.create_user_filter(expression, related)
     #
     # @param filters [Array<Hash>] Filters definition
     # @return [Array] first is list of MAQL statements
-    def self.maqlify_filters(filters, options = {})
+    def self.maqlify_filters(filters, project_users, options = {})
```

Comment:
Lint/UnusedMethodArgument: Unused method argument - `project_users`. If it's necessary, use `_` or `_project_users` as an argument name to indicate that it won't be used.